content/en/getting_started/prometheus.md

```diff
@@ -0,0 +1,158 @@
+  - prometheus_url: '<PROMETHEUS_HOST>:<PROMETHEUS_PORT>/<PROMETHEUS_ENDPOINT>'
+    namespace: '<METRICS_NAMESPACE_PREFIX_FOR_DATADOG>'
+    metrics:
+      - '<METRIC_TO_FETCH>: <NEW_METRIC_NAME>'
+```
+
+[1]: https://docs.datadoghq.com/agent/guide/agent-configuration-files/?tab=agentv6#agent-configuration-directory
+[2]: /agent/guide/agent-commands/#start-stop-and-restart-the-agent
+[3]: https://github.com/DataDog/integrations-core/blob/master/prometheus/datadog_checks/prometheus/data/conf.yaml.example
+{{% /tab %}}
+{{% tab "Docker" %}}
+
+The Agent detects if it's running on Docker and automatically searches all containers labels for the Datadog-Prometheus labels. Autodiscovery expects labels to look like these examples, depending on the file type:
+
+**Dockerfile**
+```
+LABEL "com.datadoghq.ad.check_names"='["prometheus"]'
+LABEL "com.datadoghq.ad.init_configs"='["{}"]'
+LABEL "com.datadoghq.ad.instances"='["{\"prometheus_url\":\"http://%%host%%:<PROMETHEUS_PORT>/<PROMETHEUS_ENDPOINT> \",\"namespace\":\"<METRICS_NAMESPACE_PREFIX_FOR_DATADOG>\",\"metrics\":[\"<PROMETHEUS_METRIC_TO_FETCH>: <DATADOG_NEW_METRIC_NAME>\"]}"]'
+```
+
+**docker-compose.yaml**
+```yaml
+labels:
+  com.datadoghq.ad.check_names: '["prometheus"]'
+  com.datadoghq.ad.init_configs: '["{}"]'
+  com.datadoghq.ad.instances: '["{\"prometheus_url\":\"http://%%host%%:<PROMETHEUS_PORT>/<PROMETHEUS_ENDPOINT> \",\"namespace\":\"<METRICS_NAMESPACE_PREFIX_FOR_DATADOG>\",\"metrics\":[\"<PROMETHEUS_METRIC_TO_FETCH>: <DATADOG_NEW_METRIC_NAME>\"]}"]'
+```
+
+**docker run command**
+```
+-l com.datadoghq.ad.check_names='["prometheus"]' -l com.datadoghq.ad.init_configs='["{}"]' -l com.datadoghq.ad.instances='["{\"prometheus_url\":\"http://%%host%%:<PROMETHEUS_PORT>/<PROMETHEUS_ENDPOINT> \",\"namespace\":\"<METRICS_NAMESPACE_PREFIX_FOR_DATADOG>\",\"metrics\":[\"<PROMETHEUS_METRIC_TO_FETCH>: <DATADOG_NEW_METRIC_NAME>\"]}"]'
+```
+
+{{% /tab %}}
+{{% tab "Kubernetes" %}}
+
+As all [Datadog Agent integrations][1], the Datadog-Prometheus Integration can be configured using [Autodiscovery][2]. This allows you to send a given container/pod exposed Prometheus metrics to Datadog by applying the following `annotations` to it:
+
+```
+# (...)
+metadata:
+#(...)
+  annotations:
+      ad.datadoghq.com/<CONTAINER_IDENTIFIER>.check_names: |
+        ["prometheus"]
+      ad.datadoghq.com/<CONTAINER_IDENTIFIER>.init_configs: |
+        [{}]
+      ad.datadoghq.com/<CONTAINER_IDENTIFIER>.instances: |
+        [
+          {
+            "prometheus_url": "http://%%host%%:<PROMETHEUS_PORT>/<PROMETHEUS_ENDPOINT> ",
+            "namespace": "<METRICS_NAMESPACE_PREFIX_FOR_DATADOG>",
+            "metrics": ["<PROMETHEUS_METRIC_TO_FETCH>: <DATADOG_NEW_METRIC_NAME>"]
+          }
+        ]
+spec:
+  containers:
+    - name: '<CONTAINER_IDENTIFIER>'
+```
+
+[1]: /getting_started/integrations
+[2]: /agent/autodiscovery
+{{% /tab %}}
+{{< /tabs >}}
+
+With the following configuration placeholder values:
```

Comment:
the `host` tab has `<PROMETHEUS_HOST>` as well

Reply:
It can connect to a remote host, what I meant is that it should be listed here, next to `<PROMETHEUS_PORT>`

Reply:
but then it wouldn't fit the other desc.. 🤔 .. i'm going to use `localhost` :)